packages/mobile/src/helpers/dto-name-modifiers/format-price/format-price.ts

```diff
@@ -1,9 +1,5 @@
 const formatPrice = (price: number | string): string => {
-  const priceAsString = price.toString();
-
-  return priceAsString.length > 3
-    ? `${priceAsString.slice(0, 2)} ${priceAsString.slice(2)}`
-    : priceAsString;
+  return price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ');
```

Comment:
What is the purpose or this?

Reply:
It will ad a space to after every 3rd number . For example 1 000, 10 000 , 100 000 000